TileFormats/Instanced3DModel/README.md

```diff
@@ -22,65 +22,180 @@ A tile is composed of a header section immediately followed by a body section.
+
+These define `up` and `right` using the oct-encoding described in
+[*A Survey of Efficient Representations of Independent Unit Vectors* by Cigolle et al.](http://jcgt.org/published/0003/02/01/).
+
+An implementation for encoding and decoding these unit vectors can be found in Cesium's
+[AttributeCompression](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/AttributeCompression.js)
+module.
+
+### Instance Position
+
+`POSITION` defines the location where an instance should be placed in Cartesian space.
+
+#### Quantized Positions
+
+If `POSITION` is not defined for an instance, its position may be stored in `POSITION_QUANTIZED` which defines the instance position relative to the quantized volume.
+If neither `POSITION` or `POSITION_QUANTIZED` are defined, the instance will not be created.
+
+A quantized volume is defined by `offset` and `scale` to map quantized positions into model space.
+
+**Figure 4**: A quantized volume based on `offset` and `scale`.
+
+![quantized volume](figures/quantized-volume.png)
+
+`offset` is stored in the global semantic `QUANTIZED_VOLUME_OFFSET`, and `scale` is stored in the global semantic `QUANTIZED_VOLUME_SCALE`.
+If those global semantics are not defined, `POSITION_QUANTIZED` cannot be used.
+
+Quantized positions can be mapped to model space using the formula:
+
+`POSITION = POSITION_QUANTIZED * QUANTIZED_VOLUME_SCALE + QUANTIZED_VOLUME_OFFSET`
 
-_TODO: create a separate Batch Table spec that b3dm, i3dm, etc. can reference, [#32](https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/32)?_
+### Instance Scaling
 
-The batch table is a `UTF-8` string containing JSON.  It immediately follows the header.  It can be extracted from the arraybuffer using the `TextDecoder` JavaScript API and transformed to a JavaScript object with `JSON.parse`.
+Scaling can be applied to instances using the `SCALE` and `SCALE_NON_UNIFORM` semantics.
+`SCALE` applies a uniform scale along all axes, and `SCALE_NON_UNIFORM` applies scaling to the `x`, `y`, and `z` axes independently.
 
-Each property in the object is an array with its length equal to `header.batchLength`.  Array elements can be any valid JSON data type, including objects and arrays.  Elements may be `null`.
+If `SCALE` and `SCALE_NON_UNIFORM` are defined for an instance, both scaling operations will be applied.
+
+### Examples
+
+#### Positions Only
+
+In this minimal example, we place 4 instances on the corners of a unit length square with the default orientation.
 
-An instance's `batchId` is used to access elements in each array and extract the corresponding properties.  For example, the following batch table has properties for two instances:
 ```json
 {
-    "id" : ["unique id", "another unique id"],
-    "displayName" : ["Tree species", "Another tree species"],
-    "yearPlanted" : [1999, 2003],
-    "location" : [{"x" : 1, "y" : 2}, {"x" : 3, "y" : 4}]
+    "INSTANCES_LENGTH" : 4,
+    "POSITION" : [
```

Comment:
Does the feature table JSON support pointing to the feature table binary yet?  I'm not asking about the general purpose glTF-like `bufferView`/`accessor`.  It would be fine if it is just for well known semantics so they are not stored as bulky arrays in JSON.